environments/prod/prod.tfvars

```diff
@@ -1773,6 +1773,14 @@ frontends = [
     ssl_mode         = "AzureKeyVault"
     certificate_name = "courttribunalfinder-service-gov-uk"
   },
+  {
+      product          = "rpts"
+      name             = "rpts"
+      mode             = "Detection"
+      custom_domain    = "www.residential-property-tribunals-service.gov.uk"
+      backend_domain   = ["firewall-prod-int-palo-prod.uksouth.cloudapp.azure.com"]
+      certificate_name = "residential-property-tribunals-service-gov-uk"
```

Comment:
why was this re-added? it should be using a managed certificate

Reply:
removed now was testing with Rees Pozzi